Jenkinsfile.nightly

```diff
@@ -103,12 +103,9 @@ pipeline {
         stage('Kubernetes tests') {
             steps {
                 parallel(
```

Comment:
If is only one please remove the parallel too.

Reply:
Done